lib/file_manipulation.rb

```diff
@@ -33,4 +33,42 @@ def players_names
     end
     all_players
   end
+
+  def players_kills
+    players = players_names
+    kills_occurrences = []
+    suicide_kills = []
+    kill_count = {}
+    File.readlines(@path_to_file).each do |line|
```

Comment:
pula uma linha pra separar bloco de codigos :)

```suggestion

    File.readlines(@path_to_file).each do |line|
```